tests/accelerator/sycl/CMakeLists.txt

```diff
@@ -4,34 +4,29 @@
 #
 # Mozilla Public License Version 2.0
 
-# Enable SYCL as a language.
-enable_language( SYCL )
-
-# Set the default SYCL compiler flags.
-include( algebra-plugins-compiler-options-sycl )
-
-if(${CMAKE_SYCL_STANDARD} LESS 20)
-   message(SEND_ERROR "CMAKE_SYCL_STANDARD=${CMAKE_SYCL_STANDARD}, but algebra-plugins requires C++>=20")
-endif()
-
 # Helper library for the SYCL tests.
 add_library( algebra_tests_sycl_common INTERFACE )
 target_include_directories( algebra_tests_sycl_common INTERFACE
    "${CMAKE_CURRENT_SOURCE_DIR}/common" )
 target_link_libraries( algebra_tests_sycl_common
-   INTERFACE GTest::gtest algebra::tests_common algebra::tests_accel_common
-             vecmem::core vecmem::sycl )
+   INTERFACE GTest::gtest algebra::tests_common algebra::tests_accel_common )
 add_library( algebra::tests_sycl_common ALIAS algebra_tests_sycl_common )
 
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fsycl")
```

Comment:
Come on... No. This is not flying like this. Why did you think that hardcoding the use of just oneAPI like this is the right move? 😕